packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/lib/base/models/errors/bad_request_error_model.dart

```diff
@@ -0,0 +1,16 @@
+part of 'error_model.dart';
+
+class BadRequestErrorModel extends ErrorModel implements L10nErrorKeyProvider {
+  BadRequestErrorModel({
+    this.message,
+    Map<String, String>? errorLogDetails,
+  }) : super(errorLogDetails);
+
+  final String? message;
+
+  @override
+  String get l10nErrorKey => I18nErrorKeys.accessDenied;
```

Comment:
Usually bad requests does not mean access denied

Reply:
Addressed!